src/crewai/telemetry/telemetry.py

```diff
@@ -163,6 +163,19 @@ def task_started(self, task: Task) -> Span | None:
                 tracer = trace.get_tracer("crewai.telemetry")
                 span = tracer.start_span("Task Execution")
 
+                created_span = tracer.start_span("Task Created")
+
+                self._add_attribute(created_span, "task_id", str(task.id))
+                self._add_attribute(
+                    created_span, "formatted_description", task.description
+                )
+                self._add_attribute(
+                    created_span, "formatted_expected_output", task.expected_output
+                )
```

Comment:
because this could include PII let's only add it to the trace if the user opt-ins using the `share_crew` option